plugin-packages/multimedia/src/audio/audio-component.ts

```
@@ -0,0 +1,103 @@
+import { effectsClass, RendererComponent, spec } from '@galacean/effects';
+import { AudioPlayer } from './audio-player';
+
+@effectsClass(spec.DataType.AudioComponent)
+export class AudioComponent extends RendererComponent {
+  audioPlayer: AudioPlayer;
+  private isVideoPlay = false;
+  private threshold = 0.03;
+
+  constructor (engine: Engine) {
+    super(engine);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)